app/assets/stylesheets/application.scss

```diff
@@ -41,7 +41,15 @@ body p {
     font-size: 20px;
     margin-top: 10px;
 }
-
+.comment-list{
+    width: 100%;
+    height: 70px;
+    border-bottom: 1px solid #e3e8ed;
+    padding: 5px;
+    color: #525d66;
+    font-size: 14px;
+    font-family: Lato,Arial,sans-serif;
```

Comment:
You are using a non default font family. Make sure to include this font with your webpage.

Reply:
Yes i have included an import statement in css file . I got this from google fonts